backend/server/importlib/backend.py

```diff
@@ -1499,39 +1499,39 @@ def processSuseProductRepositories(self, batch):
                AND repo_id = :repo_id
         """)
         _query_pr = self.dbmodule.prepare("""
-            SELECT product_id, root_product_id, repo_id FROM suseProductRepository
+            SELECT product_id, root_product_id, repo_id FROM suseProductSCCRepository
             """)
         _query_pr.execute()
-        existing_data = ["%s-%s-%s" % (x['product_id'], x['root_product_id'], x['repo_id']) for x in _query_pr.fetchall_dict() or []]
+        existing_data = ["%s-%s-%s" % (x['product_pdid'], x['root_pdid'], x['repo_pdid']) for x in _query_pr.fetchall_dict() or []]
         toinsert = [[], [], [], [], [], [], [], [], []]
         todelete = [[], [], []]
         toupdate = [[], [], [], [], [], [], [], []]
         for item in batch:
-            ident = "%s-%s-%s" % (item['product_id'], item['root_id'], item['repo_id'])
+            ident = "%s-%s-%s" % (item['product_pdid'], item['root_pdid'], item['repo_pdid'])
```

Comment:
Not sure here. It depends what are the keys in "batch"

Reply:
This is correct in this particular case.